spec/test-outputs/www-integration.out.vcl

```diff
@@ -99,6 +90,10 @@ sub vcl_recv {
   set req.backend = F_origin;
   set req.http.Fastly-Backend-Name = "origin";
 
+  # TODO: wrap in if statement for test mode %>
+  # uses the default test backend
+  set req.backend = s1;
```

Comment:
This is a test origin backend used by varnishtest - there would be an `<% if test env %>` block wrapping it here.